packages/devtools/lib/src/debugger/debugger.dart

```diff
@@ -1155,14 +1165,9 @@ class BreakpointsView implements CoreElementView {
   }
 }
 
-const int DOM_VK_RETURN = 13;
-const int DOM_VK_ESCAPE = 27;
-const int DOM_VK_PAGE_UP = 33;
-const int DOM_VK_PAGE_DOWN = 34;
-const int DOM_VK_END = 35;
-const int DOM_VK_HOME = 36;
-const int DOM_VK_UP = 38;
-const int DOM_VK_DOWN = 40;
+class NullTreeSanitizer implements html.NodeTreeSanitizer {
```

Comment:
Is a null tree sanitizer actually sanitizing?

Reply:
No, should be using the default sanitizer.